### PR TITLE
Update doc for error response in l1 tx endpoint

### PR DESCRIPTION
**Pull request by TheMMaciek**

### CODE REVIEW

File: node/l1-public-v2.yml [hunk 1]
```diff
@@ -48,14 +48,34 @@ paths:
               $ref: '#/components/schemas/Transaction'
       responses:
         200:
-          description: $DAG transaction has been received for processing
+          description: Hash of $DAG transaction that has been received for processing
```
Comment (marcinwadon) [inline node/l1-public-v2.yml:51]: It's a description of the status (endpoint), not the response. So 200 Ok means that the DAG transaction has been accepted for processing

File: node/l1-public-v2.yml [hunk 2]
```diff
@@ -48,14 +48,34 @@ paths:
...
             X-Session-Token:
               $ref: '#/components/headers/X-Session-Token'
             Request-Signature:
               $ref: '#/components/headers/Request-Signature'
+          content:
+            application/json:
+              schema:
+                type: string
+        400:
+          description: $DAG transaction rejection reason
```
Comment (marcinwadon) [inline node/l1-public-v2.yml:64]: Maybe "$DAG transaction is rejected"? As before, it's the description of the "result"

**Review by marcinwadon — CHANGES_REQUESTED** [review]

File: node/l1-public-v2.yml (hunk 1 above)
Comment (TheMMaciek) [reply node/l1-public-v2.yml:51]: Shouldn't we also describe the body somewhere? 🤔

Comment (TheMMaciek) [reply node/l1-public-v2.yml:51]: We return a string, that's a hash but we don't mention it anywhere.

File: node/l1-public-v2.yml (hunk 2 above)
Comment (TheMMaciek) [reply node/l1-public-v2.yml:64]: If that's the concept, than of course, I will adjust it.

File: node/l1-public-v2.yml (hunk 1 above)
Comment (marcinwadon) [reply node/l1-public-v2.yml:51]: Probably we can put a description for the response itself

Comment (TheMMaciek) [reply node/l1-public-v2.yml:51]: I will check it out.

**Review by marcinwadon — APPROVED** [review]

**Review by marcinwadon — APPROVED** [review]

**Pull request merged**